lib/external/keyboard_avoider/bottom_area_avoider.dart

```diff
@@ -175,7 +175,7 @@ RenderObject? findFocusedObject(RenderObject? root) {
     final config = SemanticsConfiguration();
     //ignore: invalid_use_of_protected_member
     node.describeSemanticsConfiguration(config);
-    if (config.isFocused) {
+    if (config.isFocused == true) {
```

Comment:
```suggestion
    if (config.isFocused ?? false) {
```

https://dart.dev/tools/linter-rules/use_if_null_to_convert_nulls_to_bools

https://dart.dev/tools/linter-rules/no_literal_bool_comparisons